frontend/components/model/pug/model.pug

```diff
@@ -6,11 +6,11 @@
 	close-focus-mode-button
 
 	#modelUI
-		revisions(
-			account="vm.account"
+		revisions-switch(
+			teamspace="vm.account"
```

Comment:
can we change `vm.account` to `vm.teamspace`? or is it going into a differetn migration?

Reply:
this will be changed in model migration